templates/docs/rhpam74-authoring.adoc

```diff
@@ -328,9 +328,8 @@ information.
 |`RHPAM_DRIVER` | -- | h2
 |`RHPAM_USERNAME` | -- | `${KIE_SERVER_H2_USER}`
 |`RHPAM_PASSWORD` | -- | `${KIE_SERVER_H2_PWD}`
+|`RHPAM_NONXA` | -- | `false`
```

Comment:
Just for information, you don't need to change the adoc files manually, we update it before the release (release tag creation) using a auxiliary script.